testcases/kernel/controllers/memcg/control/memcg_control_test.sh

```diff
@@ -53,6 +53,8 @@ STATUS_PIPE="$TMP/status_pipe"
 PASS=0
 FAIL=1
 
+. test.sh
+
```

Comment:
I haven't noticed that the test wasn't using the test.sh library before. Including it like this could have some side effects.

Anyway upon closer look at the testcase we will likely remove it see:

http://lists.linux.it/pipermail/ltp/2016-November/002926.html

So there is no point in fixing the test anyway.